app/3.0/start/quick-tour.md

```diff
@@ -38,15 +50,24 @@ an element name with a class, so you can add properties and methods to your cust
 element. The custom element's name **must start with an ASCII letter and
 contain a dash (-)**.
 
-<demo-tabs selected="0" name="qt-1-register" src="http://plnkr.co/edit/Q4E8zO?p=preview">
-  <demo-tab heading="custom-element.html">
-<pre><code>{{{include_file('2.0/start/samples/custom-element/custom-element.html')}}}</code></pre>
-  </demo-tab>
-  <demo-tab heading="index.html">
-<pre><code>{{{include_file('2.0/start/samples/custom-element/index.html')}}}</code></pre>
-  </demo-tab>
+<demo-tabs selected="0" name="qt-1-register" src="http://plnkr.co/edit/r1bwLm?p=preview">
+  <paper-tab slot="tabs">custom-element.js</paper-tab>
+  <div>
+
+```html
+<!-- include_file 3.0/start/samples/custom-element/custom-element.js -->
```

Comment:
Try opening the code block with "\```js" or "\```javascript" instead

Reply:
Does not fix the rendering issue.. but the js is nicely highlighted now, thanks :) 

Re-uploading. Would you mind taking another look?